index.js

```diff
@@ -0,0 +1,162 @@
+import addToLocalStorage from './modules/addLocal.js';
+import time from './modules/dateTime.js';
+
+const bookInput = document.querySelector('.book-input');
+const authorInput = document.querySelector('.author-input');
+const submit = document.querySelector('.add');
+const booksDiv = document.querySelector('.books');
+
+// Empty Array to store the Book
```

Comment:
- [ ] Great job. Kindly remove the numerous comments so as to follow the clean code rule.